src/puppetlabs/puppetdb/status.clj

```diff
@@ -0,0 +1,39 @@
+(ns puppetlabs.puppetdb.status
+  (:require [puppetlabs.puppetdb.scf.storage-utils :as sutils]
+            [puppetlabs.puppetdb.utils :as utils]
+            [puppetlabs.puppetdb.mq :as mq]
+            [puppetlabs.puppetdb.schema :as pls]
+            [schema.core :as s]))
+
+(def status-details-schema {:maintenance_mode? s/Bool
```

Comment:
Yes, so in PE assoc in those two additional pieces, add an fn to take that info into account and we should be good to go